SiteContentDetection/GoogleAnalyticsImporter.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link    https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Plugins\GoogleAnalyticsImporter\SiteContentDetection;
+
+use Piwik\Container\StaticContainer;
+use Piwik\Nonce;
+use Piwik\Piwik;
+use Piwik\Plugins\ConnectAccounts\Strategy\Google\GoogleConnect;
+use Piwik\Plugins\GoogleAnalyticsImporter\Controller;
+use Piwik\Plugins\GoogleAnalyticsImporter\Google\Authorization;
+use Piwik\Plugins\SitesManager\SiteContentDetection\GoogleAnalytics3;
+use Piwik\Plugins\SitesManager\SiteContentDetection\GoogleAnalytics4;
+use Piwik\SiteContentDetector;
+use Piwik\View;
+
+class GoogleAnalyticsImporter extends \Piwik\Plugins\SitesManager\SiteContentDetection\SiteContentDetectionAbstract
+{
+    public static function getName(): string
+    {
+        return 'Google Analytics Import';
```

Comment:
```suggestion
        return Piwik::translate('GoogleAnalyticsImporter_AdminMenuTitle');
```

Reply:
Looks like I forgot to commit that, as I had that change done locally already 🙈